docs/UserGuide.md

```diff
@@ -327,7 +327,8 @@ Format: `rsv INDEX d/DATE t/TIME p/PAX`
 * The index **must be a positive integer** 1, 2, 3, …​
 * `DATE` **must be in yyyy-MM-dd** format.
 * `TIME` **must be in HHmm** format.
-* `PAX` **must be a positive integer**.
+* `PAX` **must be a positive integer**. 
+* Adding multiple reservations with the same name at `INDEX`, `DATE` and `TIME` is not allowed.
```

Comment:
"same name at `INDEX`" might not be very clear to the user.

perhaps "Two reservations made under the same person's `NAME` cannot have the same `DATE` and `TIME`, i.e., combination of `DATE`, `TIME` and person `NAME` of a reservation must be unique."